index.js

```diff
@@ -345,6 +357,9 @@ var MongodbDriver = Base.extend({
           case 'updateMany':
             db.collection(collection)[command](options.query, options.update, options.options, callbackFunction);
             break;
+          case 'getDbInstance':
+            prCB(null, db); // When the user wants to get the DB instance we need to retrun the promise callback, so the DB connection is not automatically closed
```

Comment:
@wzrdtales like this you mean?

Reply:
Yes, but I typo slipped in `retrun` instead of `return` :)

Reply:
@wzrdtales Good catch 😄 Corrected it